lms/static/sass/views/_shoppingcart.scss

```diff
@@ -225,7 +225,7 @@ $light-border: 1px solid $gray-l5;
     color: $dark-gray1;
   }
 
-  ul.steps {
+  ol.steps {
```

Comment:
No it doesn't. I changed it to be correct, but should have just removed it entirely.